libraries/engage/page/widgets/Video/Video.jsx

```diff
@@ -69,23 +72,32 @@ const Video = () => {
 
   return (
     <div className={classes.root}>
-      <video
-        ref={videoRef}
-        // Set play position to 0.001s to guarantee that there is always a frame shown
-        src={`${url}#t=0.001`}
-        muted={muted}
-        controls={(!autoplay || reduceMotion) ? true : controls}
-        autoPlay={reduceMotion ? false : autoplay}
-        className={classes.video}
-        preload="auto"
-        playsInline
-        loop={loop}
-        aria-hidden
-        onError={() => { setHasError(true); }}
+      <ConditionalWrapper
+        condition={!!link && !controls && !IS_PAGE_PREVIEW_ACTIVE}
```

Comment:
please discuss with the PO how to handle situations where `autoplay` is activated, but `reduceMotion` is turned on. in that situations we show controls, even if its not configured in the widget config.

navigation is disabled in preview mode, so i think the check for `IS_PAGE_PREVIEW_ACTIVE` shouldn't be necessary.

additionally this constant shouldn't be used to detect if preview mode is active. the `useWidget` hook returns a property that can be used to check if preview is active
```js
const { config, isPreview } = useWidget();
```
so if the check should still be needed, you could extend the return value of the `useVideoWidget` hook.